### PR TITLE
Update audit logs for faster querying

### PR DESCRIPTION
**Pull request by rgraber**
## Checklist

1. [ ] If you've added code that should be tested, add tests N/A
2. [ ] If you've changed APIs, update (or create!) the documentation N/A
3. [ ] Ensure the tests pass
4. [x] Make sure that your code lints and that you've followed [our coding style](https://github.com/kobotoolbox/kpi/blob/master/CONTRIBUTING.md)
5. [x] Write a title and, if necessary, a description of your work suitable for publishing in our [release notes](https://community.kobotoolbox.org/tag/release-notes)
6. [ ] Mention any related issues in this repository (as #ISSUE) and in other repositories (as kobotoolbox/other#ISSUE) N/A
7. [ ] Open an issue in the [docs](https://github.com/kobotoolbox/docs/issues/new) if there are UI/UX changes N/A

## Description

Add some indexes on audit logs for faster querying by asset, and add a new 'auth' type for access logs.

## Notes
Adds a new indexes for the AuditLog table on (metadata__asset_uid) and (metadata__asset_uid, action) as well as a new AUTH AuditAction

### CODE REVIEW

File: kobo/apps/audit_log/migrations/0007_alter_auditlog_action_and_more.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+        ),
+        migrations.AddIndex(
+            model_name='auditlog',
+            index=models.Index(models.F('metadata__asset_uid'), models.F('action'), name='audit_log_asset_action_idx'),
+        ),
+        migrations.AddIndex(
+            model_name='auditlog',
+            index=models.Index(models.F('metadata__asset_uid'), name='audit_log_asset_uid_idx'),
+        ),
+    ]
```
Comment (noliveleger) [inline kobo/apps/audit_log/migrations/0007_alter_auditlog_action_and_more.py:26]: We should use `SKIP_HEAVY_MIGRATIONS` with manual instructions for big databases. 
Index creation could lock the table for a while and would make the app unresponsive. 

Have a look at https://github.com/kobotoolbox/kpi/blob/main/kpi/migrations/0046_project_view_assets_indexes.py for an example.

**Review by noliveleger — CHANGES_REQUESTED** [review]

File: kobo/apps/audit_log/migrations/0007_alter_auditlog_action_and_more.py (hunk 1 above)
Comment (rgraber) [reply kobo/apps/audit_log/migrations/0007_alter_auditlog_action_and_more.py:26]: Do we expect the audit log tables to already be large? i thought they only had deletions in them right now

Comment (rgraber) [reply kobo/apps/audit_log/migrations/0007_alter_auditlog_action_and_more.py]: updated in any case

Comment (noliveleger) [reply kobo/apps/audit_log/migrations/0007_alter_auditlog_action_and_more.py]: Yes, it already has a lots of records.

**Review by noliveleger — APPROVED** [review]

**Review by noliveleger — APPROVED** [review]

**Pull request merged**